config/config.default.php

```diff
@@ -27,14 +32,17 @@
     'detail.count' => 6,
     'page.limit' => 25,
 
-    // Profile 1 in 100 requests.
+    // Profile x in 100 requests. (E.g. set XHGUI_PROFLING_RATIO=50 to profile 50% of requests)
     // You can return true to profile every request.
     'profiler.enable' => function() {
-        return rand(1, 100) === 42;
+        $ratio = getenv('XHGUI_PROFILING_RATIO') ?: 100;
+        return (getenv('XHGUI_PROFILING') !== false) && (mt_rand(1, 100) <= $ratio);
```

Comment:
When would this be `false`? Environment variables are string typed so `XHGUI_PROFILING=false` will result in `'false'` as a string.

Reply:
`getenv()` returns `false` if the environment var is not set. So profiling is disabled by default. If you set the environment var with any content, the profiling will be activated.

Reply:
Ok.

Reply:
Do you merge this into the master? If so, when? I need this for the company I work for.